### DISCUSSION TITLE
Logo ideas

### DISCUSSION THREAD

**Discussion by shonfeder** [informalsystems/quint#529] [Ideas]
we need a logo. let's use this discussion to brainstorm and share ideas

**Comment by shonfeder** [comment]
Something based around Newton's symbol for quintessence: 
![image](https://user-images.githubusercontent.com/19820422/213017851-d6a42a65-a501-4583-a1a6-e071129234b2.png)

> A symbol for quintessence in the works of [Isaac Newton](https://en.wikipedia.org/wiki/Isaac_Newton). Also abbreviated ⟨q̄⟩ and ⟨ē⟩.[[14]](https://en.wikipedia.org/wiki/Aether_(classical_element)#cite_note-14)

basing something around the `q` abbreviation could also be cool.

**Comment by shonfeder** [comment]
something based around the stylized `q`:

![image](https://user-images.githubusercontent.com/19820422/213018266-773699f2-9641-4cf4-8bff-5c85cecfcb41.png)

**Comment by konnov** [comment]
I think we have converged on a first version with @designsingh.
